spec/lib/ansible/runner/credential/machine_credential_spec.rb

```diff
@@ -101,6 +101,40 @@ def password_hash
 
         expect(password_hash["^SSH [pP]assword:"]).to eq(password)
       end
+
+      context "with an existing password_file" do
+        let(:ssh_unlock_key) { "^Enter passphrase for [a-zA-Z0-9\-\/]+\/ssh_key_data:" }
+        def existing_env_password_file(data)
+          cred # initialize the dir
+          File.write password_file, data.to_yaml
+        end
+
+        it "clobbers existing ssh key unlock keys" do
+          existing_data = { ssh_unlock_key => "hunter2" }
```

Comment:
ummmm... shouldn't hunter2 be ******* ?  Or do I only see it because it's *my* password, but you see *******? 😆